test/specs/rules/validate-indentation.js

```diff
@@ -156,6 +156,23 @@ describe('rules/validate-indentation', function() {
         ]);
     });
 
+    it('should not report errors for using tabs when validateIndentation is a number', function() {
+        checker.configure({ validateIndentation: 4 });
```

Comment:
Thinking about moving this functionality under a different setting. Maybe something like:
```
"validateIndentation": {
    "value": 4,
    "allowTabs": true
}
```

Thoughts?